lib/plugins/rotate_asg_instances/asg.rb

```diff
@@ -1,25 +1,42 @@
-require 'moonshot/ssh_fork_executor'
-
 module Moonshot
   class ASG # rubocop:disable Metrics/ClassLength
```

Comment:
Should we have a module RotateAsgInstances to follow the folder path and give a better namespace to this class?

Reply:
Makes sense.